lib/datadog/core/normalizer.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Datadog
+  module Core
+    module Normalizer
+      module_function
+
+      INVALID_TAG_CHARACTERS = %r{[^\p{L}0-9_\-:./]}
+      LEADING_INVALID_CHARS_NO_DIGITS = %r{\A[^\p{L}:]++}
+      LEADING_INVALID_CHARS_WITH_DIGITS = %r{\A[^\p{L}0-9:./\-]++}
+      MAX_CHARACTER_LENGTH = 200
+      MAX_BYTE_LENGTH = MAX_CHARACTER_LENGTH * 2
```

Comment:
I think we should call these
```suggestion
      MAX_BYTE_SIZE = 200
      MAX_BYTE_SIZE_BUFFER = MAX_BYTE_SIZE * 2
```
to emphasize the limit on the byte representation a string.